templates/lib/systemd/system/caddy.service.erb

```diff
@@ -0,0 +1,56 @@
+[Unit]
```

Comment:
are you interested in porting this over to an epp template? that's the new preferred way instead of erb.